src/styles/index.css

```diff
@@ -0,0 +1,228 @@
+:root {
+  --accent-color: #00acdc;
+  --main-text-color: #616070;
+  --accent-text-color: #060b35;
+  --white: #fff;
+  --border-color: #f3f3f3;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the universal selector (*) should not be used for setting 'box-sizing'. It's better to use '.class, .class *' selector.